com.tradingview.tw.yaml

```diff
@@ -0,0 +1,53 @@
+id: com.tradingview.tw
```

Comment:
did you come up with the ID or the app uses it internally? If the name of the app is `tradingview` then it should be `com.tradingview.tradingview`

Reply:
I am certain that this name is chosen because the official website is named as such, https://tw.tradingview.com/desktop/. If you think tradingview.tradingview would be better, I can modify it.

Reply:
`tw` is not the app name

Which domain do they control? `tw.tradingview.com` or `tradingview.com` ? I see both redirecting to the same thing.

If it is the first one it should be `com.tradingview.tw.tradingview`, if it is the second one, `com.tradingview.tradingview`

Reply:
I guess the `tw` just gives the Chinese locale website? Then the second one would be fine.

Reply:
ok, I will modify it. thank you for your suggests!

Reply:
31f1cea8d0ca8041a5f88a1f0f5f9e4ec44c0160  updated

Reply:
13968ecd919c25dfe7ad20a5c3ffcdb79f0ce815  change policy url from tw to www